### ISSUE TITLE
Filters on the Order page in the BO do not work on iOS default browser, or Safari on a Mac

### ISSUE THREAD

**Issue by NiftyGaloot** [openfoodfoundation/openfoodnetwork#11091]
## Description
When using filters in the back office of OFN on the Orders page none of the filters produce any results when selecting Filter Results when using an iPhone with Safari or a Mac with Safari.
This makes it impossible to filter orders on a iPhone with the default browser or on a Mac with Safari.
Hub managers have been unable to filter orders as needed or find a specific customer's order as needed without using a different browser

## Expected Behavior
When using a filter to find orders for an order cycle only orders for that should be shown.
When using a filter for a customer's name only orders for that customer should be shown.

## Actual Behaviour
When selecting Filter Results the Orders page reloads and all of the selections entered into the filters are deleted with all orders displayed and none of the filters applied

## Steps to Reproduce

1. Use an iPhone with the default browser or Mac running Safari
2. Go to the Orders page in the back office
3. Put one or more selections as filters
4. Click the Apply filters button

## Animated Gif/Screenshot
Mac OS10.13.5, Safari 13.1.2
Before: Filters entered:
![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/132571876/56b7d472-5fb7-403e-b567-88d1ecfb1171)

After: Filters cleared:
![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/132571876/a4550a74-330a-4960-9566-0227b1901de3)

## Workaround
Use a different browser

## Your Environment
iPhone 7, iPhone SE, 2010 Mac Mini with Mac OS10.13.5, Safari 13.1.2

**Comment by NiftyGaloot** [comment]
Don't know if it's related or not... A customer unable to place an order on her iphone was able to order using chrome on the iphone instead of the default browser.

**Comment by Matt-Yorkley** [comment]
Related issue: https://github.com/openfoodfoundation/openfoodnetwork/issues/10998

**Comment by RachL** [comment]
@filipefurtad0 sorry I can't check myself as I'm commenting from my phone: what are the stat for the roll-up view (all ofn instances - not just OFN UK)?

My first hunch would be to say that we are not supporting the old version of safari only the most recent one. I don't know if it's easy to add a message in that case. So I would close the issue.

**Comment by jibees** [comment]
Supporting only the most recent version of Safari also means that we aren't supporting the old version of iPhone (or mac as well) ie. the ones that cannot handle upgrade (for different reasons, but I believe because hardware is old). Because Safari comes with the OS, and one cannot upgrade Safari on an old OS (if I understood correctly).

**Comment by NiftyGaloot** [comment]
I have been getting more reports with folks having trouble ordering using
safari. Safari is the default browser on iphones and Macs. Older devices
can not upgrade to the latest version. The only work around I know is to
use a different browser such as chrome or Firefox.

**Comment by filipefurtad0** [comment]
Hey @RachL ,

> what are the stat for the roll-up view (all ofn instances - not just OFN UK)

I'm having issues setting up the same segment, for roll-up view, posted [here](https://openfoodnetwork.slack.com/archives/C014V9TCWH5/p1688566540662219).

Overall (roll-up), we have about 1/3 Safari users - please note that this is for all pages, not just `/admin`:

![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/49817236/962c50cb-e8fd-4453-87b1-152a536736bc)

**Comment by RachL** [comment]
@NiftyGaloot do you mean reports from shoppers on the checkout? Because this issue is about admin pages.

@filipefurtad0 OK thanks I will check the segment when I'm back at computer.

**Comment by filipefurtad0** [comment]
I could not find anything related to the checkout pages, for this specific error:

![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/49817236/fcda3948-ce83-4e73-a1e2-3aa3a205781a)

This is all happening on `/admin`.

https://app.bugsnag.com/yaycode/open-food-network-js-canada/timeline?filters[event.since]=2023-06-10T06%3A43%3A31.287Z&filters[error.status]=open&filters[search][]=SecurityError%3A&filters[search][]=The&filters[search][]=operation&filters[search][]=is&filters[search][]=insecure&filters[event.before]=2023-07-05T16%3A12%3A16.953Z&sort=last_seen&pivot_tab=browser.version

Users having issues with checkout may be experiencing some other error.

**Comment by dacook** [comment]
@filipefurtad0 ,  I've created an issue for `SecurityError` here: https://github.com/openfoodfoundation/openfoodnetwork/issues/11174

Looking across a few instances it seems to have started with the current release, so might not be directly connected to this issue. But reading the comments, it sounds like there's _also_ an issue with checkout which may have been due to the latest release. We should create a new issue for that!

**Comment by filipefurtad0** [comment]
Filters work now!!

![image](https://github.com/openfoodfoundation/openfoodnetwork/assets/49817236/57d809a3-32f5-42cf-94f5-b52a3bbe3bec)

Closing :tada:
